src/hipscat/inspection/pixel_density_map.py

```diff
@@ -0,0 +1,44 @@
+"""Generate a molleview map with the pixel densities of the catalog"""
+
+import time
+
+import healpy as hp
+import numpy as np
+from matplotlib import pyplot as plt
+
+from hipscat.catalog import Catalog
+
+
+def run():
```

Comment:
This may need to be a test, or an example in the docs (thus doctested), but having a script in a library feels unusual

Reply:
Removed.